rel/overlay/etc/default.ini

```diff
@@ -318,12 +318,8 @@ compression_level = 8 ; from 1 (lowest, fastest) to 9 (highest, slowest), 0 to d
 compressible_types = text/*, application/javascript, application/json, application/xml
 
 [replicator]
-; minimum time between a replicator job restart (milliseconds)
-start_delay = 0
-; random splay time between a replicator job restart (milliseconds)
-start_splay = 0
-; Maximum replicaton retry count can be a non-negative integer or "infinity".
-max_replication_retry_count = 10
+; Random jitter applied on replication job startup (milliseconds)
+;startup_jitter = 5000
```

Comment:
default.ini should list out *ALL* of the defaults in the code. It should act as a secondary specification/documentation source, supplementing the couchdb-documentation repo. Please add all config options that you have added here, and include their default values. A one-line comment before each one is nice, but not required.

Also, don't comment out the default values - match the rest of the file in this regard. I spent hours and hours getting this file accurate, let's keep it that way :) 👍